src/node.h

```diff
@@ -513,6 +514,9 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
 #define NODE_MODULE_CONTEXT_AWARE_BUILTIN(modname, regfunc)           \
   NODE_MODULE_CONTEXT_AWARE_X(modname, regfunc, NULL, NM_F_BUILTIN)   \
 
+#define NODE_MODULE_CONTEXT_AWARE_INTERNAL(modname, regfunc)  \
```

Comment:
The `\` at the end here should be aligned with the one on the line below.

Reply:
should be fixed